src/Controllers/FrontEnd/StudentApplicationFormPageController.php

```diff
@@ -34,8 +34,12 @@ public function __construct(ApplicationFormModel $model, PhpRenderer $view)
     public function __invoke(RequestInterface $request, ResponseInterface $response, array $args): ResponseInterface
     {
         $data = [
-            'backgroundInfo' => $this->model->getBackgroundInfo(),
-            'hearAbout' => $this->model->getHearAbout(),
+            'data' => [
+                'backgroundInfo' => $this->model->getBackgroundInfo(),
+                'hearAbout' => $this->model->getHearAbout(),
+                'Cohorts' => $this->model->getCohorts(),
```

Comment:
Not sure if this is because of how it is written elsewhere but think 'cohorts' and 'gender' would be more consistent casing